src/java.desktop/share/classes/com/sun/beans/introspect/MethodInfo.java

```diff
@@ -25,18 +25,34 @@
 
 package com.sun.beans.introspect;
 
+import java.io.Closeable;
+import java.io.Externalizable;
+import java.io.Serializable;
 import java.lang.reflect.Method;
 import java.lang.reflect.Modifier;
 import java.lang.reflect.Type;
 import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.Collections;
 import java.util.Comparator;
+import java.util.HashSet;
 import java.util.List;
 
 import com.sun.beans.TypeResolver;
 import com.sun.beans.finder.MethodFinder;
 
 final class MethodInfo {
+
+    static final HashSet<Class<?>> IGNORABLE_INTERFACES = new HashSet<>(6);
```

Comment:
```suggestion
    static final HashSet<Class<?>> IGNORABLE_INTERFACES = HashSet.newHashSet(6);
```
Or even better, use Set.of instead.

Reply:
Thanks, much cleaner. Fixed in e37a1467a57.